### ISSUE TITLE
`SandboxViolation` when installing Kytos as a dependency from PyPI

### ISSUE THREAD

**Issue by erickvermot** [kytos/kytos#494]
if one inserts kytos as a requirement in a setuptools' install-requires argument, setup will abort since kytos setup.py is trying to insert files in arbitrary OS locations (/etc /var...) outside the build directory.
This happens because we are not following setuptools default procedures.
We should be moving/copying those files in a post-install script.

see https://github.com/kytos/kytos/issues/471, and https://github.com/kytos/kytos/issues/493

```
Searching for kytos-utils>=2017.1b3
Reading https://pypi.python.org/simple/kytos-utils/
Downloading https://pypi.python.org/packages/23/5b/32a2affe4b13f8de90e853066c65f2416f9940c016fbc29875ce173f2fa2/kytos-utils-2017.1b3.tar.gz#md5=67e95a90581693265e5220f0cb3006c4
Best match: kytos-utils 2017.1b3
Processing kytos-utils-2017.1b3.tar.gz
Writing /tmp/easy_install-8zys8oab/kytos-utils-2017.1b3/setup.cfg
Running kytos-utils-2017.1b3/setup.py -q bdist_egg --dist-dir /tmp/easy_install-8zys8oab/kytos-utils-2017.1b3/egg-dist-tmp-yc03lqrl
error: Setup script exited with error: SandboxViolation: mkdir('/etc/skel/kytos/napp-structure/username', 511) {}

The package setup script has attempted to modify files on your system
that are not within the EasyInstall build area, and has been aborted.

This package cannot be safely installed by EasyInstall, and may not
support alternate installation locations even if you run its setup
script by hand.  Please inform the package's author and the EasyInstall
maintainers to find out if a fix or workaround is available.
```

**Comment by hdiogenes** [comment]
Superseded by #922.

**Comment by hdiogenes** [comment]
Steps to reproduce:

If you add kytos as a dependency to a NApp:

```
setup(name='kytos_of_core',
      install_requires=[
          'kytos',
          'python-openflow'
      ],
```
And try to install it:
`python setup.py develop`

You get the `SandboxViolation` error quoted in the issue description.

**Comment by hdiogenes** [comment]
Fixed in #1042 / #1053.
